locations/spiders/bmw_es.py

```diff
@@ -0,0 +1,26 @@
+import scrapy
+
+from locations.dict_parser import DictParser
+
+
+class BeerBmwEsSpider(scrapy.Spider):
+    name = "bmw_es"
+    item_attributes = {
+        "brand": "bmw",
+        "brand_wikidata": "Q26678",
+    }
+    allowed_domains = ["bmw.es"]
+    start_urls = [
+        "https://www.bmw.es/c2b-localsearch/services/api/v4/clients/BMWSTAGE2_DLO/ES/pois?brand=BMW_BMWI_BMWM&cached=off&category=BM&country=ES&language=es&lat=0&lng=0&maxResults=700&showAll=true&unit=km"
+    ]
+
+    def parse(self, response):
+        results = response.json()["data"]["pois"]
```

Comment:
It looks like this comes back as XML, not JSON.

Reply:
Maybe try sending an `accept: application/json` header in the initial request?

Reply:
Yea same result